<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="operation_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Operation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="algorithm_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="operation_comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>encrypt without keyfile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>decrypt without keyfile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>encrypt with keyfile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>decrypt with keyfile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>generate and save key to file</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="algoritm_comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>XOR_Cipher</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AES128</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyfile_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>key/keyfile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyfile_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBlackCrypt">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuBlackCrypt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionf">
   <property name="text">
    <string>f</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
